<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainJogoVelha</class>
 <widget class="QMainWindow" name="MainJogoVelha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jogo da Velha</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="jogoVelha">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>122</width>
      <height>122</height>
     </rect>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="jogadorVez">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>5</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Jogador da vez:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgVez">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>23</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jogadorVencedor">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>76</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vencedor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgVencedor">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>92</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReiniciar">
    <property name="title">
     <string>Jogo</string>
    </property>
    <addaction name="actionReiniciar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionExibir_ajuda"/>
   </widget>
   <addaction name="menuReiniciar"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReiniciar">
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionExibir_ajuda">
   <property name="text">
    <string>Exibir ajuda...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
